package.json

```diff
@@ -21,15 +21,15 @@
     "prebuild": "rimraf lib",
     "build": "babel --quiet --out-dir lib src",
     "postbuild": "npm run copy-metafiles",
-    "copy-metafiles": "for DIR in memo-parser resolvers/node resolvers/webpack utils; do cp LICENSE .npmrc \"${DIR}/\"; done",
+    "copy-metafiles": "node -e \"require('babel-register'); require('./scripts/copyMetafiles');\"",
```

Comment:
```suggestion
    "copy-metafiles": "node --require=babel-register ./scripts/copyMetafiles",
```

Reply:
Oh - awesome, didn't know this is possible 😅